src/server/RHUI.py

```diff
@@ -0,0 +1,24 @@
+#
+# RHUI Helper
+# Provides abstraction for user interface
+#
+
+import logging
+logger = logging.getLogger(__name__)
+
+class RHUI():
+    def __init__(self):
+        self.pilot_attributes = []
+
+    def register_pilot_attribute(self, name, label, fieldtype="text"):
+        self.pilot_attributes.append(PilotAttribute(name, label, fieldtype="text"))
+        return True
+
+    def get_pilot_attributes(self):
+        return self.pilot_attributes
```

Comment:
Why a getter if the attributes are this trivially usable? In general, getters are frowned upon in Python.